code/style.css

```diff
@@ -1,4 +1,342 @@
+  animation-direction: alternate;
 }
+
+
+@keyframes gradient {
+  0% {
+      background-position: 0%;
+  }
+  100% {background-position: 100%;}
+}
+
+
+/* HEADER */
+
+.millenial-header {
+  font-size: 60px;
+  color: #E05297;
+  font-family: 'Rampart One', cursive;
+}
+
+
+/*  NAVIGATION BAR  */
+
+ul {
+  font-family: roboto, sans-serif;
+  max-width: 320px;
+  list-style-type: none;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+  margin-bottom: 20px;
+}
+
+li {
+  float:left;
+}
+
+li a {
+  display: block;
+  color: #E05297;
+  font-family: 'Rampart One', cursive;
+  font-size: 24px;
+  text-align: center;
+  padding: 5px 5px;
+  text-decoration: none;
+}
+
+/*  MAIN CONTENT (CONTAINER)  */
+
+.grid-container {
+  color: #E05297;
+  margin-left: 10px;
+  margin-bottom: 30px;
+  width: 280px;
+  display: grid;
+  row-gap: 15px;
+}
+
+/* NEWS (GRID ITEMS) */
+
+.grid-item, .grid-item-big {
+  background-color: #FCEEF5;
+  grid-column: span 1;
+  border-radius: 2%;
+  margin-top: none;
+  padding: 2% 10% 10% 10%;
+}
+
+.grid-item:hover, .grid-item-big:hover {
+  box-shadow:  0 0 1px 1px #E05297;
+}
+
+/* TEXTS */
+
+h1 {
+  font-size: 20px;
+}
+
+h2 {
+  font-size: 16px;
+}
+
+h4 {
+  font-size: 12px;
+}
+
+p {
+  font-size: 16px;
+  display: inline-block;
+}
+
+/* LINKS */
+
+a {
+  color: #E36BAE;
+  text-decoration: none;
+}
+
+
+a:hover {
+  text-decoration: underline;
+  
+}
+
+/* IMAGES */
+
+img {
+  border-radius: 2%;
+  width: 100%;
+}
+
+img:hover {
+    transform: scaleX(-1);
+}
+
+/* FOOTER */
+
+.millenial-footer {
+  color: #E05297;
+  padding: 2%;
+  font-family: 'Rampart One', cursive;
+  font-size: 16px;
+  margin-bottom: 10px;
+}
+
+.socials {
+  font-family: roboto, sans-serif;
+  font-size: 40px;
+  display: block;
+  margin-bottom: 20px;
+}
+
+
+/* ------------------------------ MEDIA QUERIES ------------------------------------------- */
+
+
+/* ---------------------------- MEDIUM: TABLET SIZE -------------------------------------- */
+
+@media only screen and (max-width: 1100px) and (min-width: 670px) {
+
+  body {
+      margin-top: 10px;
+      margin-left: 60px;
+  }
+
+  ul {
+      max-width: 1200px;
+      margin-bottom: 30px;
+  }
+
+  li a {
+      font-size: 24px;
+      padding: 15px 12px;
+  }
+
+  .grid-container {
+      width: 600px;
+      grid-template-columns: 4fr 4fr;
+      gap: 15px;
+      margin-bottom: 60px;
+      
+  }
+
+  .millenial-header {
+      font-size: 100px;
+      margin-bottom: 0;
+  }
+  
+  .grid-item, .grid-item-big {
+      
+      border-radius: 2%;
+      font-size: 16px;
+      margin-top: none;
+      padding: 5%;
+  }
+
+  h2 {
+      font-size: 20px;
+  }
+
+  h4 {
+      font-size: 12px;
+  }
+  
+  .millenial-footer {
+      margin-bottom: 50px;
+      font-size: 24px;
+  }
+
+  .socials {
+    display: inline-block;
+    margin-right: 30px;
+  }
+  
+}
+
+/* ----------------------- LARGE: DESKTOP SIZE -------------------------------- */ 
```

Comment:
I'm not yet very 100% clear of the media queries myself, but I'm a little curious about what happens between the different queries. When I noticed that a specific width between 1100px and 1400 the smaller news items change the shape and becomes more of a vertical rectangle. was it to fit the width of the big news grid better in the view? :)

Reply:
Yes, I just thought it looked better layout-wise (honestly had to go back and inspect this myself 😅).